index.html

```diff
@@ -0,0 +1,42 @@
+<h1>📘 The Code Magazine</h1>
```

Comment:
You're missing the root html structure. 
<doctype>
<HTML>
<head> & <body>

Reply:
added html structure
![image](https://github.com/shurachen23/Tekwill-Course/assets/121938064/bf77f668-7b3c-4ecf-baf4-40160dbcc82b)